pages/index.js

```diff
@@ -1,7 +1,34 @@
+import data from "@/data/tasks.json";
+import { useState } from "react";
+import { TaskCard } from "@/components/TaskCard";
 export default function HomePage() {
+  const [visible, setVisible] = useState(true);
+  console.log(data);
+  //диструктуризация
+  const { projects } = data;
   return (
-    <div>
-      <h1>Hello from Next.js</h1>
+    <div className="App">
+      <header>
+        <button>+</button>
+        <a href="/new">Link</a>
```

Comment:
You need to decide how you want to navigate to the `new` page.
Either use the `button` with an `onClick` and the `router` to progessively route the user. Or use the `Link` component from Next for routing between pages.

Reply:
OK